vyper/exceptions.py

```diff
@@ -125,6 +125,10 @@ def __str__(self):
         return f"{self.message}\n{annotation_msg}"
 
 
+class VyperException(_VyperException):
```

Comment:
why?

Reply:
so that `VyperInternalException` can also inherit the functionality from `_VyperException` (without subclassing `VyperException` directly, which would break some try/except clauses).

Reply:
ah okay. can it have a different name? possible `_BaseVyperException` or something?

Reply:
ah good call -- done in https://github.com/vyperlang/vyper/pull/3406/commits/ae5793ab7b2f3581b5a2aafc032d60de15a05cff